### ISSUE TITLE
Symfony 4 compatibility

### ISSUE THREAD

**Issue by martijnhartlief** [anyx/LoginGateBundle#9]
```Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - anyx/login-gate-bundle 0.4 requires symfony/symfony ~2.8|~3.0 -> satisfiable by symfony/symfony[2.8.x-dev, 3.0.x-dev, 3.1.x-dev, 3.2.x-dev, 3.3.x-dev, 3.4.x-dev ...
```

It probably can be fixed with ~2.8|~3.0|~4.0

**Comment by anyx** [comment]
I am closing this after merge #10

**Comment by Nikeev** [comment]
Hello,

I tried to install bundle on Symfony 4, but an error still exists.
It seems that there is no more symfony/symfony package in Symfony 4 with new Symfony Flex.
https://symfony.com/doc/current/setup/flex.html
So default Symfony 4 composer.json has 
```
"conflict": {
      "symfony/symfony": "*"
},
```

**Comment by anyx** [comment]
Hi @Nikeev,

Just released 0.6 version with changed dependencies.
Thank you for report
